pkg/cloud/aws/services/ec2/subnets_test.go

```diff
@@ -149,6 +153,10 @@ func TestReconcileSubnets(t *testing.T) {
 								Name:   aws.String("tag-key"),
 								Values: []*string{aws.String("kubernetes.io/cluster/test-cluster")},
 							},
+							{
```

Comment:
Any reason we need to actually compare the arguments here instead of using gomock.Any()?

Reply:
We could, but in this case for example, we do want to make sure the filters are actually in place. I think we should use `Any` as some sort of last resort, otherwise it'll be hard to tell what unit tests are testing, thoughts?